include/geometric_shapes/shape_extents.h

```diff
@@ -35,16 +35,17 @@
 #ifndef GEOMETRIC_SHAPES_SHAPE_EXTENTS_
 #define GEOMETRIC_SHAPES_SHAPE_EXTENTS_
 
-#include <shape_msgs/SolidPrimitive.h>
-#include <shape_msgs/Mesh.h>
+#include "shape_msgs/msg/solid_primitive.hpp"
```

Comment:
You shouldn't change `<...>` to quotes `"..."`. They have a different meaning!
Quotes should be used for local files only.
This applies to all files.